apps/src/applab/GoogleChart.js

```diff
@@ -178,15 +188,25 @@ class PieChart extends GoogleChart {
    * @override
    */
   verifyData_(data, columns) {
-    super.verifyData_(data, columns);
-
-    if (columns.length > 2) {
-      this.warn('Too many columns for pie chart; only using the first 2.');
-    }
+    super.verifyData_(data, columns, /* minColumns */ 2, /* maxColumns */ 2);
```

Comment:
If you feel a need to annotate these parameters anytime we use them, maybe it's time to switch to named parameters, especially with shorthand property names to keep it readable.

```js
super.verifyData_({data, columns, minColumns: 2, maxColumns: 2});
```